### DISCUSSION TITLE
Playground for real-time speech2text web apps

### DISCUSSION THREAD

**Discussion by saharmor** [openai/whisper#258] [Show and tell]
I wrote a full-stack playground for developers to build and launch Whisper-powered web apps.
The next step would be to use socket.io to stream mic input and incrementally send it to the backend for transcription.

Open to any feedback and contributions! 

https://github.com/saharmor/whisper-playground

https://user-images.githubusercontent.com/6180201/194215774-b34c8e10-a6c6-47ae-9ca0-f1f76ef3269d.mp4

**Comment by harish-garg** [comment]
this is awesome. 

What kind of min server reqs are needed for running the backend? is GPU required?

**Reply by saharmor** [reply]
Nope, you can run the playground using CPU
